packages/template-retail-react-app/app/pages/cart/index.jsx

```diff
@@ -75,7 +75,10 @@ const Cart = () => {
                 status: 'success',
                 action: (
                     <Button variant="link" onClick={() => navigate('/account/wishlist')}>
-                        View
+                        <FormattedMessage
+                            defaultMessage="View"
+                            id="cart.link.added_to_wishlist.view_wishlist"
+                        />
```

Comment:
This change is not seen when I added an item to Wishlist from the PLP page. I think there's another component that has a similar View button.

Reply:
I've updated the individual buttons with the same hard-coded message for now. I'll extract it to a helper in the follow-up PR.